### ISSUE TITLE
Track crying and bedtime

### ISSUE THREAD

**Issue by danepowell** [babybuddy/babybuddy#412]
During sleep training, it's helpful to track how long your child cries throughout the night, as well as when you put her down to sleep, regardless of whether she actually sleeps right away or not. I'd like to be able to log crying as an activity similar to feeding, sleep, and tummy time. And I'd like to be able to log "bedtime"... somehow 😄 I'm not exactly sure how that fits in the data model.

This might be covered by https://github.com/babybuddy/babybuddy/issues/217, though I'm not sure if the concept of "events" is the same as "activities".

**Comment by cdubz** [comment]
For bedtime you can log sleep entries — they have a start and end timer or you can just start a timer and the entry from the timer.

Crying is a tougher sell. I do think that is the kind of thing #217 is meant to address. We also have #396 coming along nicely and tagging could be used to keep tabs on crying time via notes. What are kind of data are you looking to get?

_P.S. I was convinced I recognized your name and eventually made the connection — the org I work for uses Acquia and I’m fairly active in the Drupal community as well (as “wells”). Any way just to say 👋_

**Comment by danepowell** [comment]
Thanks for pointing out #396, tagging could really help with this. As long as I have some structured way to categorize an activity, i.e. tagging a sleep activity as "crying" or "bedtime", I can probably filter it in InfluxDB and Grafana (which pull Baby Buddy data via Home Assistant).

I'll close this issue for now while I meditate on this, thanks for the ideas.

_Hah, now that you mention it I thought I recognized you as well. Turns out we [co-authored an Acquia Connector commit](https://www.drupal.org/project/acquia_connector/issues/3100935) 😄 The open-source world is a small one it seems. 👋 _
